markdown/events/2022/bytesize-nascent.md

```diff
@@ -23,3 +23,107 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[14:06](https://www.youtube.com/watch?v=chayGGPTnfM&t=846)
+This is calling the entirety of the transcript though, just to note up there on the left... oh... I think I missed HOMER. So I'll just talk a little bit about HOMER then. Without it (the slides). It uses a little bit more naive of a peak calling method. It's just looking for the transcript and the difference in the peak and itself. On those, it was released in 2010 out of the glass lab, and it was maintained by Chris Brenner. That was what we originally used in our paper. It works pretty well. The problem is it was made in a land before Docker. So it has a couple of problems with the way that it wants to pull in the references for you. But I finally realized you can just pass a FASTA in. It's like one line in the documentation and that works amazing. We just run that on everything because if you're going to run GroHMM and wait a couple hours, 20 minutes with HOMER, you might as well get some results on that as well. So again, I missed the slide on that.
+
+[15:18](https://www.youtube.com/watch?v=chayGGPTnfM&t=918)
+Let's now jump into PINTS identification. This is a new assay that just came out in 2022, and it's very exciting. I just left this in up at the top in Figure A. This is just also illustrating the difficulty in reproducing these, and this is on the exact same data sets. You can see the difference in the HOMER and the GroHMM results and just how much they vary and just a slight tweaking a tool. You'd expect maybe better performance, but you wouldn't expect completely different results based on what tool you're using. Down here at the bottom, this is the PINTS identification method. Just in a rudimentary way, it works very similarly to MACS2. This is a potential true peak based on the density of this, and it's very easy to pick out. It does some algorithms, picks up the local background noise from these, and these are the light blue. You can see in the purple from those, that's then a potential peak that it needs to test and see, is that actually a peak or is it just more noise from the assay itself? What PINTS is doing is really picking up these TSS start sites. As you can see from the read pile up here, it's just picking up the TSS site rather than the entirety of the transcript, which might actually lead out all the way along here.
```

Comment:
```suggestion
Let's now jump into PINTS identification. This is a new assay that just came out in 2022, and it's very exciting. I just left this in up at the top in Figure A. This is just also illustrating the difficulty in reproducing these, and this is on the exact same data sets. You can see the difference in the HOMER and the GroHMM results and just how much they vary by just a slight tweaking of a tool. You'd expect maybe better performance, but you wouldn't expect completely different results based on what tool you're using. Down here at the bottom, this is the PINTS identification method. Just in a rudimentary way, it works very similarly to MACS2. This is a potential true peak based on the density of this, and it's very easy to pick out. It does some algorithms, picks up the local background noise from these, and these are the light blue. You can see in the purple from those, that's then a potential peak that it needs to test and see, is that actually a peak or is it just more noise from the assay itself? What PINTS is doing is really picking up these TSS start sites. As you can see from the read pile up here, it's just picking up the TSS site rather than the entirety of the transcript, which might actually lead out all the way along here.
```